features/org.wso2.carbon.ldap.server.server.feature/resources/conf/identityPerson.ldif

```diff
@@ -160,6 +166,6 @@ objectClasses: ( 1.3.6.1.4.1.37505.1.101
     SUP scimPerson
     STRUCTURAL
     MAY  ( primaryChallenges $ challenges $ firstChallenge $ secondChallenge $ oneTimePassword $ accountLock $ temporaryEmail $ recoveryEmail $ passwordChangeRequired $ passwordTimestamp $ temporaryLock $ lastFailedAttemptTime $ failedLoginAttempts $ lastLogonTime $ unlockTime $ verifyEmail $ askPassword $ forcePasswordReset $ failedRecoveryAttempts $ primaryChallengeQuestion $ emailVerified $ challengeQuestionUris $ 
-    $ lastLoginTime $ lastPasswordUpdate $ phoneVerified $ accountDisabled $ preferredChannel $ lockedReason $ pendingEmailAddress)
+    $ lastLoginTime $ lastPasswordUpdate $ phoneVerified $ accountDisabled $ preferredChannel $ lockedReason $ pendingMobileNumber )
```

Comment:
Why are we removing pendingEmailAddress?

Reply:
Addressed in https://github.com/wso2-extensions/identity-userstore-ldap/pull/39/commits/f586cf881757961d0c0e00e9af6e3495e09c1a05